pnpm-lock.yaml

```diff
@@ -28130,7 +28129,7 @@ packages:
       ignore: 5.3.0
     dev: true
 
-  /eslint-plugin-i/2.29.0_j7h7oj6rrhtikhzta4fgkou42e:
+  /eslint-plugin-i/2.29.0_p6unqcplhqas2g2cyfcrkqcjbu:
```

Comment:
suspect but okay resolution change
Are the ts-jest changes already dealt with in other PRs? Nice to see deduplication.